TicTacToe/TicTacToe.py

```diff
@@ -0,0 +1,152 @@
+def get_board_size():
+    """ Board must be squared and and odd (with one center """
+    while True:
+        try:
+            print("Enter board Size (odd number)")
+            board_size = int(input())
+        except ValueError:
+            print("Board size must be a Number")
+            continue
+        if(board_size % 2 == 0 ):
+            print("Number must be odd")
+        # is odd number
+        else:
+            break
+    return board_size
+
+
+def init_board(board_size):
+    """ print blanked board with '0' in each cell """
+    board = [[0 for cell in range(0,board_size)] for row in range(0,board_size)]
+    return board
+
+
+def print_board_state(board):
+    """ Draw the board after each turn """
+    for row in board:
+        print(" -"*len(board))
+        print("|", end='')
+        for cell  in row:
+            if cell  == 0:
+                print(" ", end='')
+            else:
+                print(cell , end='')
+            print("|",end='')
+        print("")
+        print(" -"*len(board))
+
+
+def choose_player(board):
+    """ axiom Player X always starts """
+    sum_X_on_board = 0
+    sum_O_on_board = 0
+    for row in board:
+        sum_X_on_board += row.count("X")
+        sum_O_on_board += row.count("O")
+    # out of blanked cells is a Tie
+    if((sum_X_on_board+sum_O_on_board) == (len(board)*len(board))):
+        print("Game Done it's a Tie")
+        sign = "Tie"
+    elif(sum_X_on_board <= sum_O_on_board or sum_X_on_board == 0):
+        print("Player X turn:")
+        sign = "X"
+    else:
+        print("Player O turn:")
+        sign = "O"
+    return sign
+
+
+def play_random_move(board, sign):
+    """ choose cell randomly from all the empty places """
+    from random import randrange
```

Comment:
usually we import all of the necessary modules in the start of the file. it is easier to understand what is required for your program that way.